packages/wdio-applitools-service/src/index.js

```diff
@@ -0,0 +1,73 @@
+import logger from 'wdio-logger'
+import { Eyes, Target } from '@applitools/eyes.webdriverio'
+
+const log = logger('wdio-applitools-service')
+
+const DEFAULT_VIEWPORT = {
+    width: 1440,
+    height: 900
+}
+
+export default class ApplitoolsService {
+    constructor () {
+        this.eyes = new Eyes()
+    }
+
+    /**
+     * set API key in onPrepare hook and start test
+     */
+    beforeSession (config) {
+        const key = process.env.APPLITOOLS_KEY || config.applitoolsKey
+        const applitoolsConfig = config.applitools || {}
+
+        if (!key) {
+            throw new Error(`Couldn't find an Applitools private key in config nor environment`)
```

Comment:
Can we add `applitoolsKey` and `APPLITOOLS_KEY` to error message to make it more clear?
Something like `Couldn't find an Applitools 'applitoolsKey' in config nor environment 'APPLITOOLS_KEY' variable.`